src/main/java/bio/terra/app/configuration/DuosConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.app.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "duos")
+public class DuosConfiguration {
+
+  private String basePath;
+
+  public String getBasePath() {
+    return basePath;
+  }
+
+  public void setBasePath(String basePath) {
+    this.basePath = basePath;
+  }
+}
```

Comment:
I think this can become a `record`
```suggestion
public record DuosConfiguration(String basePath) {}
```

Reply:
Update, I was not able to get this to work: while reducing boilerplate code is good, I had a hard time getting the immutability of the record to play nicely with Spring Boot.  We may be missing some higher app-level annotation that could enable this (`@ConfigurationPropertiesScan`?) but I'm designating this as OOS.

Reply:
I was curious about this so I created a branch and (I think) got it working. The one thing I didn't know is that, using an older version of Spring Boot, you need to use the constructor binding annotation. See https://github.com/DataBiosphere/jade-data-repo/pull/1374

Reply:
(This is separate from the `ConfigurationPropertiesScan` change, which lets you remove `Configuration` from this class.)